File: Chapter-04-Strings-AssociativeArrays/String-Reverse/String-Reverse.py
# Implement reverseString(), given stringm returns the string with the characters in reverse order. Given "creature" return "erutaerc". Do not use built-in functions

def reverseString(string):
    result = ''

    for i in range(1, len(string)+1):
        result += string[-i]

    return result

myStr = "creature this big"
print("The original string is '{}'").format(myStr)
print("The reversed string is '{}'").format(reverseString(myStr))

File: Chapter-04-Strings-AssociativeArrays/String-Reverse/String-Reverse.js
// Implement reverseString(), given stringm returns the string with the characters in reverse order. Given "creature" return "erutaerc". Do not use built-in functions

function reverseString(str){
  var result = ''
  for (let i = str.length - 1; i >= 0; i--){
    result += str[i]
  }

  return result
}

var myStr = "creature this big"
console.log(`The original string is '${myStr}'`)
console.log(`The reversed string is '${reverseString(myStr)}'`)
